app/views/modals/loginDialog.html

```diff
@@ -4,9 +4,16 @@
 </div>
 <div class="modal-body">
   <div align="center">
-    <p>You must login to continue.<br><br>Log in with</p>
-    <p><a ng-href="{{apiUrl}}auth/facebook/authorization"><img src="/img/signin-facebook-grey.png"></a></p>
-    <p>or</p>
-    <p><a ng-href="{{apiUrl}}auth/relay/login?logoutCallbackUrl={{apiUrl}}auth/relay/logout"><img src="/img/signin-relay-grey.png"></a></p>
+    <p class="lead spacing-below-xs">
+      Sign in with one of the following services:
+    </p>
+    <p>
+      <a class="btn btn-subtle btn-spacing" ng-href="{{apiUrl}}auth/facebook/authorization" title="Facebook"><img src="/img/signin-facebook.png"></a>
+      <a class="btn btn-subtle btn-spacing" ng-href="{{apiUrl}}auth/relay/login?logoutCallbackUrl={{apiUrl}}auth/relay/logout" title="Relay"><img src="/img/signin-relay.png"></a>
+    </p>
+    <p>
+      <span class="text-muted">or</span>
+      <a class="logo ng-scope" href="#/" ng-click="close()">Return to the Welcome Page</a>
```

Comment:
This should say "Go to Welcome Page" or something like that.  Most of the time the users will not becoming from there.